tests/test_adapt.py

```diff
@@ -71,32 +72,34 @@ def inline_old_identity_twice_graph(old_identity):
     return results(final=z).with_opset(("ai.onnx", 17))
 
 
-@pytest.fixture
-def old_squeeze_graph(old_squeeze):
-    class Squeeze11(StandardNode):
-        @dataclass
-        class Attributes(BaseAttributes):
-            axes: AttrInt64s
+class Squeeze11(StandardNode):
+    @dataclass
+    class Attributes(BaseAttributes):
+        axes: AttrInt64s
 
-        @dataclass
-        class Inputs(BaseInputs):
-            data: Var
+    @dataclass
+    class Inputs(BaseInputs):
+        data: Var
 
-        @dataclass
-        class Outputs(BaseOutputs):
-            squeezed: Var
+    @dataclass
+    class Outputs(BaseOutputs):
+        squeezed: Var
 
-        op_type = OpType("Squeeze", "", 11)
+    op_type = OpType("Squeeze", "", 11)
 
-        attrs: Attributes
-        inputs: Inputs
-        outputs: Outputs
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
 
-    def squeeze11(_data: Var, _axes: Iterable[int]):
+    @classmethod
```

Comment:
Why does this need to be a class method? Usually our constructor functions are free functions.